src/RequestInsurance/ViewComponents/EditApprovalsStatus.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Cego\RequestInsurance\ViewComponents;
+
+use Illuminate\View\View;
+use Illuminate\View\Component;
+use Illuminate\Contracts\View\Factory;
+use Cego\RequestInsurance\Models\RequestInsuranceEdit;
+
+class EditApprovalsStatus extends Component
+{
+    /**
+     * Holds the RequestInsurance instance
+     *
+     * @var RequestInsuranceEdit $requestInsuranceEdit
+     */
+    public $requestInsuranceEdit;
+
+    /**
+     * PrettyJson constructor.
+     *
+     * @param RequestInsuranceEdit $requestInsuranceEdit
+     *
+     * @throws JsonException
+     */
+    public function __construct(RequestInsuranceEdit $requestInsuranceEdit)
+    {
+        $this->requestInsuranceEdit = $requestInsuranceEdit;
+    }
+
+    /**
+     * Gets the boot strap color of the status
+     *
+     * @return string
+     */
+    public function statusColor()
+    {
+        return $this->requestInsuranceEdit->approvals()->count() >= $this->requestInsuranceEdit->required_number_of_approvals ? 'success' : 'warning';
+    }
+
+    /**
+     * Gets the text of the status
+     *
+     * @return string
+     */
+    public function statusText()
+    {
+        return sprintf('%d/%d', $this->requestInsuranceEdit->approvals()->count(), $this->requestInsuranceEdit->required_number_of_approvals);
```

Comment:
```suggestion
        return sprintf('%d/%d', $this->requestInsuranceEdit->approvals->count(), $this->requestInsuranceEdit->required_number_of_approvals);
```

Gør du sådan her begge steder, så laver du kun denne query én gang, istedet for flere gange pr. request :smile: